lucetc/src/decls.rs

```diff
@@ -285,7 +320,7 @@ impl<'a> ModuleDecls<'a> {
                     def: GlobalDef::new(init_val),
                 }
             };
-            globals.push(GlobalSpec::new(g_variant, None));
```

Comment:
This is the big change for global exports - lucetc was just not passing export names along for globals. Because they can be more than one name (one global exported under multiple names), I switched it to a Vec as well